src/NLog/Config/LoggingConfiguration.cs

```diff
@@ -316,7 +324,9 @@ public void AddRule(LogLevel minLevel, LogLevel maxLevel, Target target, string
         }
 
         /// <summary>
-        /// Add a rule with min- and maxLevel.
+        /// Add a rule with <paramref name="minLevel"/> and <paramref name="maxLevel"/>. The rule will NOT have a name.<br/>
+        /// If a name is required, use <see cref="LoggingRule(string)"/> instead and add the created rule to the
```

Comment:
The LoggingRule-name is an exotic feature, that is only relevant for special users that wants to special changes to NLog-configuration at runtime. Think normal users will wonder what is the meaning of rule-name? Is a rule-name needed? What happens if no rule-name is provided? I prefer that xml-comments doesn't mention all the stuff that are not assigned.